EventFilter/Utilities/plugins/testGlobalNumbers.cc

```diff
@@ -22,177 +24,191 @@
 #include <time.h>
 
 
-using namespace edm;
-using namespace std;
-
 namespace test{
 
   static const unsigned int GTEVMId= 812;
   static const unsigned int GTPEId= 814;
-  class GlobalNumbersAnalysis: public EDAnalyzer{
-  private:
-  public:
-    GlobalNumbersAnalysis(const ParameterSet& pset){
+  class GlobalNumbersAnalysis: public edm::EDAnalyzer {
```

Comment:
If this file is not moved to the `test` directory, the inheritance from EDAnalyzer should be changed to `edm::one::EDAnalyzer<>` to quiet the static analyzer warning.